dhall/man/dhall.1

```diff
@@ -0,0 +1,100 @@
+.\" Automatically generated by Pandoc 2.9.2
```

Comment:
My intuition is that a generated file like this shouldn't be tracked in git. A script that generates this file would be nice to have though!

Reply:
I've added a script! I think it's good to track in git since users probably don't want to download pandoc simply to get pandoc? Plus this prevents uploading to hackage without the manpage!

Reply:
Don't worry, I can take care of further automation here, so this is already good